### ISSUE TITLE
User interests quick search

### ISSUE THREAD

**Issue by divanov11** [divanov11/Mumble#196]
When user is on their home feed they should be able to simply click a tag and automatically be take to the search page with the form pre-filled, pulling up any post, article or user that contains that particular keyword.

I will take care of this functionality. Just need to update the API a bit to handle this..

![image](https://user-images.githubusercontent.com/33843378/113965510-5cc8d200-97e2-11eb-9992-17486b060dc5.png)

**Comment by divanov11** [comment]
Thank you @codyseibert , I'm such a noob sometimes lol

**Comment by codyseibert** [comment]
I don't think @divanov11 has worked on this feature, so I'm going to open it back up for anyone to work on.

**Comment by ishitb** [comment]
@codyseibert Hi. if the issue is still open I'd like to work on it.
I have a few questions though,
- The only feature to be added is "Click the tag, redirect to search page", right?
- I'm not sure where these tags are getting added from, I think they're dummy for now?
- On the search page, the Article section seems to be broken. It doesn't show anything, not even a 404
Thank you!

**Comment by codyseibert** [comment]
@ishitb i think we just need an endpoint to search by tags, such as /api/tags?Name=javascript.  The frontend will have to hook up to it later.

**Comment by codyseibert** [comment]
Right, click the tag and redirect to the search page with the tag name you clicked.

I don't think we have an endpoint yet, so I'll probably add one called /api/tags?name=JavaScript where you can pass the name and get all the users and mumbles with those tags.

I'll let you know when that endpoint is created

**Comment by ishitb** [comment]
> Right, click the tag and redirect to the search page with the tag name you clicked.
> 
> I don't think we have an endpoint yet, so I'll probably add one called /api/tags?name=JavaScript where you can pass the name and get all the users and mumbles with those tags.
> 
> I'll let you know when that endpoint is created

Alright thank you so much!
